clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -218,6 +218,13 @@ New checks
   Detects C++ code where a reference variable is used to extend the lifetime
   of a temporary object that has just been constructed.
 
+- New :doc:`readability-avoid-return-with-void-value
+  <clang-tidy/checks/readability/avoid-return-with-void-value>` check.
+
+  Complains about statements returning expressions of type ``void``. It can be
```

Comment:
Consider simplifying description of check make to something like "Finds return statements with void values used within functions with a void result type."